e2e/build.rs

```diff
@@ -0,0 +1,6 @@
+extern crate build_deps;
+
+fn main() {
+    build_deps::rerun_if_changed_paths("testdata/*.sql").unwrap();
```

Comment:
Why is this necessary? To make `cargo test` re-run tests when SQL changes? Does it get cached otherwise?

Reply:
This is necessary because the test cases are autogenerated from the `testdata` directory with a macro. If you make changes to the test code in the `src`, cargo knows that it needs to recompile. But without this `build.rs`, cargo doesn't know that it needs to recompile when you make changes to the `testdata` contents.